test/book/test.jl

```diff
@@ -9,21 +9,23 @@ isdefined(Main, :FakeTerminals) || include(joinpath(pkgdir(REPL),"test","FakeTer
 @testset "OscarBookExamples" verbose=true begin
 
   broken = [
-            # Something broken in Oscar
-            "specialized/breuer-nebe-parker-orthogonal-discriminants/expl_syl.jlcon",
-
             # non-stable:
             "specialized/joswig-kastner-lorenz-confirmable-workflows/versioninfo.jlcon",
 
             # backtrace err
             "introduction/introduction/julia.jlcon",
             "introduction/introduction/julia2.jlcon",
             "introduction/introduction/julia3.jlcon",
-
-            #  broken with recent changes ... to be fixed on master soon
-            "cornerstones/number-theory/galoismod.jlcon",
            ]
   skipped = [
+             # Something broken, probably needs some updated GAP packages
+             "specialized/breuer-nebe-parker-orthogonal-discriminants/expl_syl.jlcon",
+
+             # needs new hecke release
+             "cornerstones/number-theory/galoismod.jlcon",
```

Comment:
I just tagged a new Hecke version (https://github.com/JuliaRegistries/General/pull/104799), which should make this unncessary.